src/stan-users-guide/linear_regression_1/regression_1.stan

```diff
@@ -0,0 +1,13 @@
+data {
+  int<lower=0> N;
+  vector[N] x;
+  vector[N] y;
+}
+parameters {
+  real alpha;
+  real beta;
+  real<lower=0> sigma;
+}
+model {
+  y ~ normal(alpha + beta*x, sigma);
```

Comment:
Inconsistent!

Reply:
Follow the guidelines and use spaces around all operators, including `=` and `*` here. 

Also, this is going to need to have priors if it's going to be the kind of reference model we can't point people to.  We're going to have to rewrite the manual so that it starts with more complicated best practices so that we don't publish models we don't want people to use.

Or, this is going to have to have a huge warning spattered across it saying "DO NOT USE---FOR PEDAGOGICAL PURPOSES ONLY; SEE <PATH> FOR REPLACEMENT"

Reply:
> Also, this is going to need to have priors

Good catch.